acceptance/acceptance_test.go

```diff
@@ -393,14 +394,36 @@ func CopyDir(src, dst string, inputs, outputs map[string]bool) error {
 }
 
 func ListDir(t *testing.T, src string) ([]string, error) {
+	// exclude folders in .gitignore from comparison
+	ignored := []string{
+		"\\.ruff_cache",
+		"\\.venv",
+		".*\\.egg-info",
+		"__pycache__",
+		// depends on uv version
+		"uv.lock",
+	}
+
```

Comment:
The approach we use is to clean up unnecessary files at the end of script.

In general, it is interesting to know which test case create which files.

Reply:
Please keep these test-scoped: https://github.com/databricks/cli/pull/2215